projects.js

```diff
@@ -0,0 +1,178 @@
+    description2: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    image: 'images/work/snapshoot.png',
+    live_version: 'https://batoolfatima2135.github.io/Portfolio/',
+    source: 'https://github.com/batoolfatima2135/Portfolio',
+  },
+  {
+    title: ' Keeping track of hundreds of components',
+    description1: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    description2: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    image: 'images/work/snapshoot.png',
+    live_version: 'https://batoolfatima2135.github.io/Portfolio/',
+    source: 'https://github.com/batoolfatima2135/Portfolio',
+  },
+  {
+    title: ' Keeping track of hundreds of components',
+    description1: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    description2: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    image: 'images/work/snapshoot.png',
+    live_version: 'https://batoolfatima2135.github.io/Portfolio/',
+    source: 'https://github.com/batoolfatima2135/Portfolio',
+  },
+  {
+    title: ' Keeping track of hundreds of components',
+    description1: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    description2: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.",
+    technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    image: 'images/work/snapshoot.png',
+    live_version: 'https://batoolfatima2135.github.io/Portfolio/',
+    source: 'https://github.com/batoolfatima2135/Portfolio',
+  },
```

Comment:
- [ ] [OPTIONAL] Please update each project in the array with unique data. This way it is easy to tell which project was clicked.